src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -190,7 +193,31 @@ private String getOdoCommand() throws IOException {
           command = ProgressManager.getInstance().run(new Task.WithResult<String, IOException>(null, "Downloading Odo", true) {
             @Override
             public String compute(@NotNull ProgressIndicator progressIndicator) throws IOException {
-              OkHttpClient client = new OkHttpClient();
+              OkHttpClient client;
```

Comment:
Should be refactored as a separate method.

Reply:
Good point, will do tomorrow from home with my own github user.
Also, this class deserves a nice cleanup IMO, do you mind if I do it in another commit?

Reply:
I prefer to keep different concerns in different PRs so I would prefer another PR.
I you use a different GitHub id, don't forget the DCO (sign) header as this is required for us to accept contributions